packages/babel-parser/src/parse-error/standard-errors.js

```diff
@@ -132,6 +132,7 @@ export default (_: typeof toParseErrorCredentials) => ({
   IncompatibleRegExpUVFlags: _(
     "The 'u' and 'v' regular expression flags cannot be enabled at the same time.",
   ),
+  InvalidArguments: _("Invalid Arguments."),
```

Comment:
The error message is a bit vague. We can provide more informative errors like we did for setters:

https://github.com/babel/babel/blob/2c7e9f79bb8c28c73ecc52c53fa1bc3a25c21b63/packages/babel-parser/src/parse-error/standard-errors.js#L48-L51